formlibrary/models/distribution.py

```diff
@@ -3,9 +3,10 @@
 
 from datetime import datetime
 from django.db import models
-from workflow.models import Program, ProjectAgreement, Office, Province
+from workflow.models import ProjectAgreement, Province
```

Comment:
I believe the key model you need from workflow is `Program` and `ProjectAgreement` but `Province` can be removed.